emcc.py

```diff
@@ -1854,12 +1854,12 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
       src = open(final).read()
       final = final + '.modular.js'
       f = open(final, 'w')
-      f.write('var ' + shared.Settings.EXPORT_NAME + ' = function(Module) {\n')
-      f.write('  Module = Module || {};\n')
+      f.write('var ' + shared.Settings.EXPORT_NAME + ' = function(' + shared.Settings.EXPORT_NAME + ') {\n')
+      f.write('  ' + shared.Settings.EXPORT_NAME + ' = ' + shared.Settings.EXPORT_NAME + ' || {};\n')
       f.write('\n')
       f.write(src)
       f.write('\n')
-      f.write('  return Module;\n')
+      f.write('  return ' + shared.Settings.EXPORT_NAME + ';\n')
```

Comment:
Not sure about that because Closure will change `var Module` to `var b` or something and then the code in the middle will not assign the exported name to `Module`.